### PR TITLE
Refine `Theme` documentation to make it more factual

### PR DESCRIPTION
**Pull request by YuriSizov**
Some of the statements were incorrect or misleading, and some just didn't follow a good writing style. I've tried to touch up every description in the `Theme` class to make it correct.

### CODE REVIEW

File: doc/classes/Theme.xml [hunk 1]
```diff
@@ -71,142 +79,150 @@
...
 			</description>
 		</method>
 		<method name="get_color" qualifiers="const">
 			<return type="Color" />
 			<argument index="0" name="name" type="StringName" />
 			<argument index="1" name="theme_type" type="StringName" />
 			<description>
-				Returns the [Color] at [code]name[/code] if the theme has [code]theme_type[/code].
+				Returns the [Color] property defined by [code]name[/code] and [code]theme_type[/code], if it exists.
+				Returns the default color value, if the property doesn't exist. Use [method has_color] to check for existence.
```
Comment (Calinou) [inline doc/classes/Theme.xml:100]: Remove the comma before `if` to sound more natural:

```suggestion
				Returns the default color value if the property doesn't exist. Use [method has_color] to check for existence.
```

File: doc/classes/Theme.xml [hunk 2]
```diff
@@ -71,142 +79,150 @@
...
 			</description>
 		</method>
 		<method name="get_constant" qualifiers="const">
 			<return type="int" />
 			<argument index="0" name="name" type="StringName" />
 			<argument index="1" name="theme_type" type="StringName" />
 			<description>
-				Returns the constant at [code]name[/code] if the theme has [code]theme_type[/code].
+				Returns the constant property defined by [code]name[/code] and [code]theme_type[/code], if it exists.
+				Returns [code]0[/code], if the property doesn't exist. Use [method has_constant] to check for existence.
```
Comment (Calinou) [inline doc/classes/Theme.xml:122]:
```suggestion
				Returns [code]0[/code] if the property doesn't exist. Use [method has_constant] to check for existence.
```

File: doc/classes/Theme.xml [hunk 3]
```diff
@@ -71,142 +79,150 @@
...
 			</description>
 		</method>
 		<method name="get_font" qualifiers="const">
 			<return type="Font" />
 			<argument index="0" name="name" type="StringName" />
 			<argument index="1" name="theme_type" type="StringName" />
 			<description>
-				Returns the [Font] at [code]name[/code] if the theme has [code]theme_type[/code].
+				Returns the [Font] property defined by [code]name[/code] and [code]theme_type[/code], if it exists.
+				Returns the default theme font, if the property doesn't exist and the default theme font is set up (see [member default_font]). Use [method has_font] to check for existence of the property and [method has_default_font] to check for existence of the default theme font.
```
Comment (Calinou) [inline doc/classes/Theme.xml:144]:
```suggestion
				Returns the default theme font if the property doesn't exist and the default theme font is set up (see [member default_font]). Use [method has_font] to check for existence of the property and [method has_default_font] to check for existence of the default theme font.
```

File: doc/classes/Theme.xml [hunk 4]
```diff
@@ -71,142 +79,150 @@
...
 			</description>
 		</method>
 		<method name="get_font_size" qualifiers="const">
 			<return type="int" />
 			<argument index="0" name="name" type="StringName" />
 			<argument index="1" name="theme_type" type="StringName" />
 			<description>
-				Returns the font size at [code]name[/code] if the theme has [code]theme_type[/code].
+				Returns the font size property defined by [code]name[/code] and [code]theme_type[/code], if it exists.
+				Returns the default theme font size, if the property doesn't exist and the default theme font size is set up (see [member default_font_size]). Use [method has_font_size] to check for existence of the property and [method has_default_font_size] to check for existence of the default theme font.
```
Comment (Calinou) [inline doc/classes/Theme.xml:161]:
```suggestion
				Returns the default theme font size if the property doesn't exist and the default theme font size is set up (see [member default_font_size]). Use [method has_font_size] to check for existence of the property and [method has_default_font_size] to check for existence of the default theme font.
```

File: doc/classes/Theme.xml [hunk 5]
```diff
@@ -71,142 +79,150 @@
...
 			</description>
 		</method>
 		<method name="get_icon" qualifiers="const">
 			<return type="Texture2D" />
 			<argument index="0" name="name" type="StringName" />
 			<argument index="1" name="theme_type" type="StringName" />
 			<description>
-				Returns the icon [Texture2D] at [code]name[/code] if the theme has [code]theme_type[/code].
+				Returns the icon property defined by [code]name[/code] and [code]theme_type[/code], if it exists.
+				Returns the engine fallback icon value, if the property doesn't exist. Use [method has_icon] to check for existence.
```
Comment (Calinou) [inline doc/classes/Theme.xml:190]:
```suggestion
				Returns the engine fallback icon value if the property doesn't exist. Use [method has_icon] to check for existence.
```

File: doc/classes/Theme.xml [hunk 6]
```diff
@@ -71,142 +79,150 @@
...
 		</method>
 		<method name="get_stylebox" qualifiers="const">
 			<return type="StyleBox" />
 			<argument index="0" name="name" type="StringName" />
 			<argument index="1" name="theme_type" type="StringName" />
 			<description>
-				Returns the [StyleBox] at [code]name[/code] if the theme has [code]theme_type[/code].
-				Valid [code]name[/code]s may be found using [method get_stylebox_list]. Valid [code]theme_type[/code]s may be found using [method get_stylebox_type_list].
+				Returns the [StyleBox] property defined by [code]name[/code] and [code]theme_type[/code], if it exists.
+				Returns the engine fallback stylebox value, if the property doesn't exist. Use [method has_stylebox] to check for existence.
```
Comment (Calinou) [inline doc/classes/Theme.xml:212]:
```suggestion
				Returns the engine fallback stylebox value if the property doesn't exist. Use [method has_stylebox] to check for existence.
```

File: doc/classes/Theme.xml [hunk 7]
```diff
@@ -215,116 +231,121 @@
 			<argument index="1" name="name" type="StringName" />
 			<argument index="2" name="theme_type" type="StringName" />
 			<description>
-				Returns the theme item of [code]data_type[/code] at [code]name[/code] if the theme has [code]theme_type[/code].
-				Valid [code]name[/code]s may be found using [method get_theme_item_list] or a data type specific method. Valid [code]theme_type[/code]s may be found using [method get_theme_item_type_list] or a data type specific method.
+				Returns the theme property of [code]data_type[/code] defined by [code]name[/code] and [code]theme_type[/code], if it exists.
+				Returns the engine fallback icon value, if the property doesn't exist. Use [method has_theme_item] to check for existence.
```
Comment (Calinou) [inline doc/classes/Theme.xml:235]:
```suggestion
				Returns the engine fallback icon value if the property doesn't exist. Use [method has_theme_item] to check for existence.
```

**Review by Calinou — DISMISSED** [review]

**Comment by pycbouh** [comment]
Commas gone, build is done!

**Review by mhilbrunner — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**